fedgov-cv-message-builder/src/main/java/gov/usdot/cv/msg/builder/IntersectionSituationDataBuilder.java

```diff
@@ -393,35 +423,50 @@ public List<GeometryContainer> buildGeometryContainers(IntersectionInputData isd
 					}
 				}
 			}
+
+			
 		}
 
-		// Setting the Approach Geometry Layer
-		approachGeometryContainer.setGeometryContainerID(GeometryContainer.APPROACH_GEOMETRY_LAYER_ID);
-		approachGeometryContainer.setApproachGeometryLayer(approachGeometryLayer);
+		// Check if approachGeometryLayer approachGeomApproachSet is empty
```

Comment:
I think for this comment, its better to say: Check if ... has values,
because if it is not empty you will copy values.
Similarly for all the other comments checking if a set is empty